github-actions/src/main/scala/org/typelevel/sbt/gha/GenerativePlugin.scala

```diff
@@ -614,7 +614,6 @@ ${indent(jobs.map(compileJob(_, sbt)).mkString("\n\n"), 1)}
             key -> values.take(1) // we only want the primary value
       }
 
-      val keys = "scala" :: additions.keys.toList.sorted
```

Comment:
unused, so began failing after re-enabling the warnings.

Reply:
Also nice!